.circleci/config.yml

```diff
@@ -516,3 +494,12 @@ workflows:
             branches:
               only:
                 - raft-tdp-main
+
+  dev-deployment:
```

Comment:
This workflow handles both Terraform changes and deploying the applications to the dev space and is only run when the relevant pipeline parameters are provided.